web/index.html

```diff
@@ -2859,16 +2833,6 @@ <h3>Summary of CMSSW_12_0_X(PAT_MINIAOD) <font size="2em"><a target='_blank' hre
 				</li>
 			</ul>
 		
-			<li>
-			<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/results/comp_igprof/CMSSW_12_0_0_pre4_step4_cpu_endjob.html" title="CPU compare">[CPU compare]</a>
-			</li>
-			<li>
-			<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/results/comp_igprof/CMSSW_12_0_0_pre4_step4_mem_live.1.html" title="MEM compare">[MEM ig1 compare]</a>
-			</li>
-			<li>
-			<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/results/comp_igprof/CMSSW_12_0_0_pre4_step4_mem_live.99.html" title="MEM compare">[MEM ig99 compare]</a>
-			</li>
-	
 			<li>Circle (Pie chart):
 			<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/circles/piechart.php?local=false&dataset=CMSSW_12_0_0_pre4%2Fslc7_amd64_gcc900%2F23434.21%2Fstep4_circles&resource=time_real&colours=default&groups=package&threshold=0" title="Circle">[Circle]</a>
```

Comment:
BTW, this and other links are broken
```suggestion
			<a href="http://cms-reco-profiling.web.cern.ch/cms-reco-profiling/circles/piechart.php?local=false&dataset=CMSSW_12_0_0_pre4%2Fslc7_amd64_gcc900%2F23434.21%2Fstep4_circles&resource=time_real&colours=default&groups=packages&threshold=0" title="Circle">[Circle]</a>
```
the default in the circles is apparently `hlt` and while it still kind of works for step3, it does not for step4.